test/test.js

```diff
@@ -52,284 +58,236 @@ describe('YeldaChat', () => {
       assert.typeOf(yeldaChat.init, 'function', 'init function exists')
     })
 
-    it('should return null if assistantId or assistantSlug are not passed', () => {
+    it('should webChatContainer is null if assistantId or assistantSlug are not passed', async () => {
       const mockData = {
         'locale': 'fr_FR'
       }
 
-      const response = yeldaChat.init(mockData)
-      assert.typeOf(response, 'null', 'init function returns null')
-    })
-
-    it('should return nothing(undefined) if required data passed', () => {
-      const mockData = {
-        'assistantSlug': 'sodebo',
-        'assistantId': '12345678'
-      }
-
-      const response = yeldaChat.init(mockData)
-      assert.typeOf(response, 'undefined', 'init function returns undefined')
```

Comment:
don't we miss the success case ? 
    it('should set webChatContainer if required data passed', () => {